packages/react-native-app/screens/Account.tsx

```diff
@@ -32,11 +30,11 @@ export default function Account() {
                 </Button>
             </View>
             <View style={styles.separator}></View>
-            <Button onPress={() => connector.killSession()}>
+            {/* <Button onPress={() => connector.killSession()}>
```

Comment:
You can use the provider to disconnect

`const { provider } = useWeb3Modal();`

`<Button onPress={() => provider?.disconnect()}>`

Reply:
Yeah I was wondering the same, how to disconnect thanks for answering it here!